docs/operating-system.md

```diff
@@ -4,24 +4,41 @@
 
 ### Cloud provider
 
-|   | Ubuntu | Container Linux | CentOS |
-|---|---|---|---|
-| AWS | ✓ | ✓ | ✓ |
-| Openstack | ✓ | ✓ | ✓ |
-| Digitalocean  | ✓ | ✓ | ✓ |
-| Google Cloud Platform | ✓ | ✓ | x |
-| Hetzner | ✓ | x | ✓ |
-| Linode | ✓ | x | x |
+|   | Ubuntu | Container Linux | CentOS | RHEL | SLES |
+|---|---|---|---|---|---|
+| AWS | ✓ | ✓ | ✓ | ✓ | ✓ |
+| Azure | ✓ | ✓ | ✓ | x | x |
+| Digitalocean  | ✓ | ✓ | ✓ | x | x |
+| Google Cloud Platform | ✓ | ✓ | x | x | x |
+| Hetzner | ✓ | x | ✓ | x | ✓ |
+| Packet | ✓ | ✓ | ✓ | x | x |
+| Openstack | ✓ | ✓ | ✓ | ✓ | ✓ |
```

Comment:
Openstack doesn't support SLES

Reply:
well, nothing prevents from using SLES, you mean that it is not covered by tests?

Reply:
It is not covered by the tests and we don't have a prepared image, with cloud-init support added. As the requirement was to run it on one cloud provider only. AWS runs the e2e tests for SLES

Reply:
ok